### PR TITLE
Make helpers support RSpec

### PR DESCRIPTION
**Pull request by rubendinho**
Fixes https://github.com/bullet-train-co/magic_test/issues/7 since the change in https://github.com/bullet-train-co/magic_test/commit/ff8cb144927611349bf7ce14a1d9f632faf536cf didn't make it to the `ok` method.  

Also moved the other non-public method to `private`.  

I am now able to call `ok` but am getting a different error: 
image

### CODE REVIEW

File: lib/magic_test/support.rb [hunk 1]
```diff
@@ -102,5 +86,29 @@ def magic_test
...
+          i += 1
+          last_block.unshift(history_lines.last(i).first)
+        end
+      end
+      return last_block
+    end
+
+    # TODO this feels like it's going to end up burning people who have other support files in `test` or `spec` that don't include `helper` in the name.
+    def get_last_caller(caller)
+      binding.pry
```
Comment (andrewculver) [inline lib/magic_test/support.rb:110]: @rubendinho I'll remove this before merging.

File: lib/magic_test/support.rb [hunk 2]
```diff
@@ -102,5 +86,29 @@ def magic_test
...
+      caller select { |s| s.include?("/test/") || s.include?("/spec/") }.reject { |s| s.include?("helper") }
```
Comment (andrewculver) [inline lib/magic_test/support.rb:111]: No caller.select required here? I didn't know that.

**Comment by andrewculver** [comment]
@rubendinho So sorry! I didn't see this was marked as draft and I made some changes to try and prep for a merge. 😭

File: lib/magic_test/support.rb (hunk 2 above)
Comment (rubendinho) [reply lib/magic_test/support.rb]: must have been a typo

**Comment by rubendinho** [comment]
> @rubendinho So sorry! I didn't see this was marked as draft and I made some changes to try and prep for a merge. 😭

No worries at all - thanks for doing that.  I'm a bit stuck with the capybara error and am not sure when i'll be able to dig deeper.

**Comment by rubendinho** [comment]
@andrewculver since the Capybara error I'm getting is unrelated, I say we merge this.  I will re-open a new issue if I'm still having problems.

**Comment by andrewculver** [comment]
@rubendinho Agreed. I think we're good here for now! Keep us posted on the other issue!

**Pull request merged**